cmd/main.go

```diff
@@ -162,7 +158,17 @@ func main() {
 			log.Fatal("failed to create client:", err)
 		}
 		generator = job.New(log.WithField("pkg", "k8s-job-wrapper"), kclient, Options.JobConfig)
+
+		cfg, cerr := clientcmd.BuildConfigFromFlags("", "")
+		if cerr != nil {
+			log.Fatalf("Failed to create kubernetes cluster config")
+		}
+		k8sClient := kubernetes.NewForConfigOrDie(cfg)
```

Comment:
isn't it the same as https://github.com/openshift/assisted-service/pull/279/files#diff-3002d695552ec7e7ebbaaf04a21d3439R143 ?

Reply:
@filanov what line do u mean?

Reply:
`var kclient client.Client`

Reply:
nope, looks like i can't use it

Reply:
it is implemented in internal code.